WPFHospitalEditor/Controller/MapObjectController.cs

```diff
@@ -9,25 +9,25 @@ public class MapObjectController : IMapObjectController
     {
         private readonly IMapObjectService IMapObjectService = new MapObjectService(new MapObjectRepository());
 
-        public List<MapObject> getAllMapObjects()
+        public List<MapObject> GetAllMapObjects()
         {
-            return IMapObjectService.getAllMapObjects();
+            return IMapObjectService.GetAllMapObjects();
         }
-        public MapObject update(MapObject mapObject)
+        public MapObject Update(MapObject mapObject)
         {
-            return IMapObjectService.update(mapObject);
+            return IMapObjectService.Update(mapObject);
         }
-        public List<MapObject> getOutterMapObjects()
+        public List<MapObject> GetOutterMapObjects()
         {
-            return IMapObjectService.getOutterMapObjects();
+            return IMapObjectService.GetOutterMapObjects();
         }
-        public MapObject findMapObjectById(int id)
+        public MapObject FindMapObjectById(int id)
         {
-            return IMapObjectService.findMapObjectById(id);
+            return IMapObjectService.FindMapObjectById(id);
         }
```

Comment:
You introduce a new category of method (Find). It would be better to use already existing categories of methods such as Get or Search

Reply:
Renamed.